simSPI/fov.py

```diff
@@ -0,0 +1,249 @@
+"""Helper functions for tem.py to define grid FOV for displaying particles."""
```

Comment:
(in case nobody knows the answer, I think I intended it to mean "Field Of View" - thank you guys for putting up with my messy notebooks and making such a nice piece of code!)